rule-src/rule-en.md

```diff
@@ -2,268 +2,271 @@
-Example 2: When Sorai and Midorino, Midorino and Tsuge are couples and Sorai and Tsuge is not a couple, Sorai-Midorino-Tsuge is not a Yuri Polygamy.
+Example 2: When Sorai and Midorino, Midorino and Tsuge are couples and Sorai and Tsuge are not a couple, Sorai-Midorino-Tsuge is not a Yuri Polygamy.
 Example 3: Murafuji and Akane, Murafuji and Kuroki, Murafuji and Shirakaba, Akane and Kuroki, Akane and Shirakaba, Kuroki and Shirakaba are couples, Murafuji-Akane-Kuroki-Shirakaba forms Yuri Polygamy.
 
-Note that already-coupled girl cannot confess to any other girl. Yuri Polygamy cannot be formed only by Confessions.
+Note that already-coupled girls cannot confess to any other girl. Yuri Polygamy cannot be formed only by Confessions.
 ```
 
-When Yuri Polygamy is formed, the game ends and proceed to determinate winner. Winner is determined by sum of Support Points assigned to girls pairs included in Yuri Polygamy group.
-Tie-break is also determined by revealed Control Points same as Fatal Couple's case. Less revealed player is winner.
+When Yuri Polygamy is formed, the game ends and proceeds to winner determination. The winner is determined by the sum of Support Points assigned to the Girl Pairs included in the Yuri Polygamy group.
+Tie-breaking is also determined by revealed Control Points, the same as in the Fated Couples case. The player with fewer revealed Control Points is the winner.
 
 ## Game end by 9th turn
 
-When both Fatal Couple and Yuri Polygamy is not born at the end of 9th turn, winner is determined by existing couple(s). Count Support Points (and Control Points to tie-break) same as Fatal Couple's case.
+When neither a Fated Couple nor Yuri Polygamy are born by the end of the 9th turn, the winner is determined by existing couple(s). Count Support Points (and Control Points to tie-break), the same as the Fated Couples case.
 
 ## Misc rules
 
 - Max Favor / Discomfort level is 6.
-- Favor level token or other components may short, actual game should not be affected by constraint from physical components.
+- If you are short of Favor level tokens or other components, continue the game as though you did have these components. The game should not be affected by constraints from physical components.
 
 ## FAQ
 
-- In 1st turn, another girls pair (not an initial couple settled in Setup) became a couple. Does this couple try Kiss of Promise in 1st turn?
+- In the 1st turn, another Girl Pair (not an initial couple settled in setup) become a couple. Does this couple try the Kiss of Promise in the 1st turn?
 
-  - Yes, they try. Only initial couple formed at Setup does not try Kiss of Promise in 1st turn. The initial couple also tries Kiss of Promise like other couples after 2nd turn.
+  - Yes, they try. Only the initial couple formed at setup does not try Kiss of Promise on the 1st turn. The initial couple also tries the Kiss of Promise like other couples after the 1st turn.
```

Comment:
I was no sure whether "after" means open or close interval in English. It was critical mistranslation, thank you!